openapi/components/schemas/DataExports/resources/amlChecks.yaml

```
@@ -0,0 +1,14 @@
+description: AML Checks resource to export.
```

nitpick: capitalization 
```suggestion
description: AML checks resource to export.
```